frontend/src/components/smart/PvPArenaPreparation.vue

```diff
@@ -574,6 +574,14 @@ p, li, span {
   margin: 0 auto;
   justify-content: space-between;
 }
+@media  all and (max-width: 1057px) {
+  .mainWrapper{
+    flex-direction: column;
+  }
+  .arenaSignup .bottomList ul li{
+    white-space: inherit !important;
+  }
+}
```

Comment:
Can you try adding flex-wrap:wrap to .mainWrapper and setting a min-width / width to the character image? (currently the image just scales down when the screen gets smaller and the wrapper doesn't wrap)
This way we can avoid the media query, which will be invalid if any other component changes